### ISSUE TITLE
Fitbit Versa Lite stops retrieving data from Tomato

### ISSUE THREAD

**Issue by felixrevueltabas** [Rytiggy/Glance#85]
My partner has been using the Glance Watchface with a Versa Lite and the Tomato app (Android) as a data source for a week, and we are very impressed with it.

She has, however, encountered an issue that makes it sadly almost unusable. The Watch is  getting glucose readings normally and then randomly it stops retrieving data. This happens always with the android phone locked and with the Tomato app in the background. Once she unlocks the phone and brings Tomato to the foreground, the watch syncs and offers the current reading again. Problem is, this happens often enough that she cannot trust the reading she is getting from the Watch and has to double check with the phone, which sadly renders the watch pointless.

 Is there a setting in the Android setting/ the Fitbit app/ the Tomato app or the Glance app we are missing, that’d keep the Watch data from freezing?

Is it an issue with Bluetooth?

Cheers and thanks for the great job you’re doing

**Comment by Rytiggy** [comment]
Hey, Do you have all day sync and always connected enabled in the Fitbit app? 

Have you checked out this decision tree for syncing issues? 
https://glancewatchface.com/troubleshooting

Im happy to help trouble shoot this more

**Comment by MasterPlexus** [comment]
And also be aware that all apps which should work.also permanently in the background that they are excluded from battery optimization!

**Comment by Mararjona** [comment]
Hello, i'm mar, Félix partner. First of all, thanks for everything and your fast response. I do have all day Sync and always connected enabled in the Fitbit app and i also restart the watch but i'm still having the same problem. The thing is everything is working fine and suddenlyy, for no reason, the watch stop syncing my blood sugar levels and i need to go to Fitbit app and open It, and in a few seconds the glucose levels get update. Tomato is working property with miao miao advice, no problem with that. It,s something about syncing. It could be hours not working, during day and nite. I,d just read the decision tree but i see no answer. The app is working permanently in the background Aldo. Any other suggestion?i go on a hike for a week tomorrow and i'd really love to hace my watch working, specially at nite in mountain shelter. Thanks again

**Comment by Rytiggy** [comment]
Hey, are you keeping the Fitbit all open and not swiping it away? (Closing the app)

**Comment by Mararjona** [comment]
Yes, its open pero is not syncing continue,....rite now there is a Big
difference with tomato app and when i go to Fitbit this is what i see and
in a few seconds the sugar levels get update....

**Comment by Mararjona** [comment]
Finally my wacht is working property. I tried many different things but this is the one that worked. I have a Xiaomi redmi note7, and i've openned the automatic starting of Fitbit app. I'm súper Happy with my wacht, i've been on a hike for a few days and it has been very useful at night notifying me the sugar down without disturbing anyone at mountain shelters. Thanks a lot. I'm gonna make my donation right now.
